crates/database/src/trades.rs

```diff
@@ -45,19 +45,25 @@ LEFT OUTER JOIN LATERAL (
     AND   s.log_index > t.log_index
     ORDER BY s.log_index ASC
     LIMIT 1
-) AS settlement ON true
-JOIN orders o
-ON o.uid = t.order_uid"#;
+) AS settlement ON true"#;
+
     const QUERY: &str = const_format::concatcp!(
         COMMON_QUERY,
+        " JOIN orders o ON o.uid = t.order_uid",
         " WHERE ($1 IS NULL OR o.owner = $1)",
-        " AND ($2 IS NULL OR o.uid = $2)",
-        "UNION",
+        " AND ($2 IS NULL OR t.order_uid = $2)",
```

Comment:
nit: why this change?

Reply:
accidental leftover since i tried several different ways of merging